src/index.html

```diff
@@ -9,6 +9,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="logo.png" alt="logo">
+      </a>
+      <nav class="nav">
```

Comment:
what have you deployed?
![image](https://user-images.githubusercontent.com/86923089/208061811-e2991309-ad2c-4081-914d-dda4ac3f1619.png)

it doesn't match your code. please, deploy the latest version